src/components/custom/_dbanner.js

```diff
@@ -37,9 +37,17 @@ const BackgroundWrapper = styled(Flex)`
 
 const ImageContainer = styled.div`
     position: absolute;
-    left: 5%;
+    left: 10%;
     height: 100%;
 
+    @media (max-width: 1680px) {
+        left: 7%;
+    }
+
+    @media ${device.laptopL} {
+        left: 5%;
+    }
+
```

Comment:
Is it possible for us to use `calc()` here to calculate the left position instead of these media queries? @suthesan-deriv

Reply:
I think that's gonna be a bit tricky since there's change in the image size at those breakpoints too...unless there's a way to know the size of the image at those breakpoints and pass it into calc?